onecore_maintenance_extension/views/maintenance_views.xml

```diff
@@ -222,10 +224,24 @@
                         <br/>
                     </span>
                     <span>
-                    Prioritet: <t t-esc="record.priority_expanded.value"/>
+                    Prioritet:
+                    <t t-if="record.priority_expanded.value">
+                        <t t-esc="record.priority_expanded.value"/>
+                    </t>
+                    <t t-else="">
+                        Ej angett
```

Comment:
Looks good but "Ej angett" doesn't show when I test it. 
<img width="475" alt="image" src="https://github.com/user-attachments/assets/27ea6482-6696-4ad8-abcb-f25d7ef3142f">

Reply:
"Ej angett" is only shown on the cards in the Kanban view. Not sure if we should have it as an option here too?